app/v1/controllers/requestParsers/validators/validations/AmountValidation.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators.validations
+
+import v1.models.errors.{AmountFormatError, MtdError, RuleInvalidLossAmount}
+
+object AmountValidation {
+
+  def validate(amount: BigDecimal): List[MtdError] = {
+    if (amount.scale > 2) {
```

Comment:
Should this be `!= 2`? Otherwise they could send 1 or 0 decimal places and it would be successful.